README.md

```diff
@@ -36,15 +36,27 @@ Der Server bietet sowohl für die App als auch für den Bot die notwendigen API
 Der Source-Code und Dokumentation des Servers ist in diesem Repo zu finden.
 
 #### Install and Start
-Voraussetzung ist ein installiertes python 3.8 package
-- `sudo ./manage.sh install` um den server zu installieren
-  (sudo only necessary to install graphviz to display story graph)
-- `./manage.sh start` um den server zu starten
-- für weitere befehle `./manage.sh help` benutzen
+Voraussetzung ist ein Linux System mit installiertem python 3.8
+1. `./manage.sh install` um den server zu installieren
+1. `./manage.sh start` um den server zu starten
+1. für weitere befehle `./manage.sh help` benutzen
+1. siehe Bot repo für Installation und Start des Bots
+1. nach Start von Server und Bot kann die App und damit das Spiel gestartet werden
 
 #### Architektur-Überblick
 Der Server wird durch eine Flask App (`/app`) implementiert. Diese verwaltet verschiedene API-Endpunkte (`/app/routes`). Außerdem wird eine Datenbank verwaltet. Deren ORM wird in `/app/models` implementiert. Der Story-Inhalt und der Story-verwaltende Code liegt unter `/app/story`.  
 
+#### Nutzung ohne App
```

Comment:
We also need to stop the server from calling an invalid firebase url if we mock "no app gameplay".
This could be achived by overwriteing "execute_call" in `firebase_interaction.py` (inserting a return eg.)

Reply:
I haven't had any problems with this until now - and the server doesn't know if we're mocking the App or not (obviously, that's the purpose of mocking)
So I don't really see a need to act here - if you still do, please elaborate where the problem is

Reply:
The problem I see here that the firebase_url might not be set if you have no intent of using the app.
Which will lead to the server failing on assigning new tasks (as he tries to notify the app)

I guess you did not have problems because you still had the fb url set?

Reply:
I understand - so we can do a simple `if firebase_url` check, right?